.gitmodules

```diff
@@ -7,9 +7,6 @@
 [submodule "external/ThirdPartyDependencies"]
 	path = external/ThirdPartyDependencies
 	url = https://github.com/forcedotcom/SalesforceMobileSDK-iOS-Dependencies.git
-[submodule "external/MKNetworkKit"]
-	path = external/MKNetworkKit
-	url = https://github.com/forcedotcom/MKNetworkKit.git
-[submodule "libs/SalesforceNetworkSDK"]
-	path = libs/SalesforceNetworkSDK
-	url = https://github.com/forcedotcom/SalesforceNetworkSDK.git
+[submodule "external/ocmock"]
```

Comment:
Removed ```MKNetwork``` and the old ```SalesforceNetwork``` submodules. Added only ```OCMock``` as a submodule. The new ```SalesforceNetwork``` is a top level Mobile SDK resident.